From e88054ca95a3f63309d33607ef37ef63c1f7a9ca Mon Sep 17 00:00:00 2001
From: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
Date: Wed, 27 Feb 2019 09:49:48 +0100
Subject: [PATCH 2/5] kabi: cpufreq: keep min_sampling_rate in struct dbs_data

Patch-mainline: Never, kabi fix
References: bsc#1127042

Following patch removed min_sampling_rate. Keep it in the
struct dbs_data to keep kabi intact.

2d045036322c cpufreq: governor: Drop min_sampling_rate

Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/cpufreq/cpufreq_governor.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/cpufreq/cpufreq_governor.h b/drivers/cpufreq/cpufreq_governor.h
index 95f207eb820e..9a3a6d4d7a62 100644
--- a/drivers/cpufreq/cpufreq_governor.h
+++ b/drivers/cpufreq/cpufreq_governor.h
@@ -41,6 +41,7 @@ enum {OD_NORMAL_SAMPLE, OD_SUB_SAMPLE};
 struct dbs_data {
 	struct gov_attr_set attr_set;
 	void *tuners;
+	unsigned int min_sampling_rate; /* Unused. Left to keep kabi intact. */
 	unsigned int ignore_nice_load;
 	unsigned int sampling_rate;
 	unsigned int sampling_down_factor;
-- 
2.11.0

